src/clj/datahike_server/server.clj

```diff
@@ -203,7 +229,7 @@
       (handler request))))
 
 (def route-opts
-  {:reitit.middleware/transform dev/print-request-diffs ;; pretty diffs
+  {;; :reitit.middleware/transform dev/print-request-diffs ;; pretty diffs
```

Comment:
Just wondering: why was this commented out?

Reply:
For testing and dev there were too many things going on in the repl with that. Might add that through options again.